doc/release-notes/8431-versionNotes.md

```diff
@@ -0,0 +1,12 @@
+Dataverse now supports the option of adding a versionNote before/during publication of a dataset that can be used to indicate why a version was created and/or how it differs from the prior version. Whether this feature is enabled is controlled by a flag. Version notes are shown in the user interface (dataset page version table), indexed, available via the API, and have been added to the JSON, DDI, DataCite, and OAI-ORE exports.
```

Comment:
Can you please add a screenshot to the description of the UI when you click "Edit Note" not on the Versions tab?

Also, is the note only mutable when the dataset is a draft?

Reply:
![image](https://github.com/user-attachments/assets/a802f2cd-457a-4cf3-b6d3-4334a5033bcb)

The note can only be edited in the UI/changed by a non-superuser by UI or API when it's in the draft state. Superusers can use the API to change the versionNote of earlier versions.

Reply:
Thanks! I added it to the description as well, with the other images.